pkg/apis/gke.cattle.io/v1/types.go

```diff
@@ -55,6 +55,7 @@ type GKEClusterConfigSpec struct {
 	MasterAuthorizedNetworksConfig *GKEMasterAuthorizedNetworksConfig `json:"masterAuthorizedNetworks,omitempty" norman:"noupdate"`
 	Locations                      []string                           `json:"locations"`
 	MaintenanceWindow              *string                            `json:"maintenanceWindow,omitempty"`
+	Autopilot                      *bool                              `json:"autopilot,omitempty"`
```

Comment:
Add this feature to the docs issue. Need to explain details to the user about pros and cons and how you can/can’t configure.

Reply:
:+1: